proto/vulcanize/auction/v1beta1/types.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+package vulcanize.auction.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/tharsis/ethermint/x/auction/types";
+
+// Params defines the auction module parameters
+message Params {
+    option (gogoproto.goproto_stringer) = false;
+
+    // Duration of the commits phase in seconds
+    int64 commits_duration = 1;
```

Comment:
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration
Use `timestamp.Duration` instead of int64